gravitee-apim-console-webui/src/management/application/creation-ng/application-creation.component.ts

```diff
@@ -58,6 +59,13 @@ const TYPES_INFOS = {
   },
 };
 
+function toMetadataDictionary(additionalClientMetadata: Header[]) {
```

Comment:
Not sure why we pass a header array here ? even if the object as the same structure it is semantically weird

Reply:
`Header[]` comes from `gio-form-headers` component, which is reused to handle also metadata.

Reply:
ok, but it remains semantically weird ^^